tests/test_integration/test_common_layout.py

```diff
@@ -88,54 +58,72 @@ def test_components_created(
     # if the test passes unexpectedly,
     # it will fail the test suite
 )
+
+
+# Q for review: is there a way to have this with
+# one fixture only?
```

Comment:
So our dream scenario is:

* A fixture gives us a dictionary of page names and titles (all of them).
* A way to parametrise a test for every entry in the dictionary.

... right? I sortof agree that a tuple fixture per page is a bit clunky (although possibly also convenient).

I don't know how to do that. (Perhaps it's one to invoke the Stansby?)